src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -41,64 +35,17 @@ def acr_create(cmd,
                resource_group_name,
                sku,
                location=None,
-               storage_account_name=None,
                admin_enabled=False,
                default_action=None,
-               deployment_name=None):
+               tags=None):
     if default_action and sku not in get_premium_sku(cmd):
         raise CLIError(NETWORK_RULE_NOT_SUPPORTED)
 
-    if sku in get_managed_sku(cmd) and storage_account_name:
-        raise CLIError("Please specify '--sku {0}' without providing an existing storage account "
-                       "to create a managed registry, or specify '--sku Classic --storage-account-name {1}' "
-                       "to create a Classic registry using storage account `{1}`."
-                       .format(sku, storage_account_name))
-
-    if sku in get_classic_sku(cmd):
-        result = client.check_name_availability(registry_name)
-        if not result.name_available:
-            raise CLIError(result.message)
-
-        logger.warning(
-            "Due to the planned deprecation of the Classic registry SKU, we recommend using "
-            "Basic, Standard, or Premium for all new registries. See https://aka.ms/acr/skus for details.")
-        if storage_account_name is None:
-            storage_account_name = random_storage_account_name(cmd.cli_ctx, registry_name)
-            logger.warning(
-                "A new storage account '%s' will be created in resource group '%s'.",
-                storage_account_name,
-                resource_group_name)
-            LongRunningOperation(cmd.cli_ctx)(
-                arm_deploy_template_new_storage(
-                    cmd.cli_ctx,
-                    resource_group_name,
-                    registry_name,
-                    location,
-                    sku,
-                    storage_account_name,
-                    admin_enabled,
-                    deployment_name)
-            )
-        else:
-            LongRunningOperation(cmd.cli_ctx)(
-                arm_deploy_template_existing_storage(
-                    cmd.cli_ctx,
-                    resource_group_name,
-                    registry_name,
-                    location,
-                    sku,
-                    storage_account_name,
-                    admin_enabled,
-                    deployment_name)
-            )
-        return client.get(resource_group_name, registry_name)
-
-    if storage_account_name:
-        logger.warning(
-            "The registry '%s' in '%s' SKU is a managed registry. The specified storage account will be ignored.",
-            registry_name, sku)
+    if sku not in get_managed_sku(cmd):
```

Comment:
remove classic from --sku list, this check is redundant

Reply:
Scripts can still contain `Classic`. This way we can give them a proper error message instead of just invalid argument (slow deprecation).

Reply:
Sounds good, can we print the allowed values?